### DISCUSSION TITLE
Using docker-compose failing with JRE error - aarch64

### DISCUSSION THREAD

**Discussion by paulhwatson** [gnzsnz/ib-gateway-docker#134] [Q&A]
Error unpacking jar files. The architecture or bitness (32/64)
70.73 of the bundled JVM might not match your machine.

Apple silicon M1. Downloaded the latest repository and ran:
`docker compose up`

There is also a warning but I don't know if its relevant.
ib-gateway Warning Head "https://ghcr.io/v2/gnzsnz/ib-gateway/manifests/latest": denied: denied

I put my userid and password in the Yaml file for testing.

Any help would be great

**Comment by gnzsnz** [comment]
aarch64 architecture does not work.

there is an open issue for it, #81 and an experimental branch #102

the docker image runs on linux, on #102 I did some modification to have it working on aarch64 but there are many errors. in theory it should be possible, in practice we are not there yet.

**Reply by paulhwatson** [reply]
Thank you so much for that, I'll give it a go 😊
